charts/log-router/templates/daemonset.yaml

```diff
@@ -22,6 +22,9 @@ spec:
       labels:
         app: {{ template "fluentd-router.name" . }}
         release: {{ .Release.Name }}
+        {{- if .Values.podLabels}}
```

Comment:
I don't see a default value being set for `podLabels` in the values file.
Convention is for optional slice objects to add `podLabels: []` to the values file to indicate to users that it can be set.